### PR TITLE
[487] Disable DBD and provider chaser emails

### PR DESCRIPTION
**Pull request by avinhurry**
## Context

In line with the continuous applications work, we no longer need these emails.

## Changes proposed in this pull request

Disable RBD emails to training provider, ratifying provider and candidate If the `continuous_applications` feature flag is enabled.

## Guidance to review

Comment in line which requires clarification/confirmation

## Link to Trello card

https://trello.com/c/BqnlAGRA/488-ca-emails-remove-provider-make-a-decision-and-decline-by-default-emails

## Things to check

- [x] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [x] This code does not rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [x] API release notes have been updated if necessary
- [x] If it adds a significant user-facing change, is it documented in the [CHANGELOG](CHANGELOG.md)?
- [x] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

File: app/services/decline_offer_by_default.rb [hunk 1]
```diff
@@ -20,12 +20,14 @@ def call
...
+    unless application_form.continuous_applications?
+      application_choices.each do |application_choice|
+        NotificationsList.for(application_choice, event: :declined_by_default, include_ratifying_provider: true).each do |provider_user|
+          ProviderMailer.declined_by_default(provider_user, application_choice).deliver_later
+        end
       end
-    end
 
-    CandidateMailer.declined_by_default(application_form).deliver_later
+      CandidateMailer.declined_by_default(application_form).deliver_later
```
Comment (avinhurry) [inline app/services/decline_offer_by_default.rb:30]: The Trello card, only mentioned to remove the provider email, but it made sense to me to remove the provider email too. Just flagging here incase my sense makes no sense 😆

**Comment by avinhurry** [comment]
[CA emails: Remove provider 'make a decision' and 'decline by default' emails](https://trello.com/c/BqnlAGRA/488-ca-emails-remove-provider-make-a-decision-and-decline-by-default-emails)

File: app/services/decline_offer_by_default.rb (hunk 1 above)
Comment (tomas-stefano) [reply app/services/decline_offer_by_default.rb:30]: You mean remove the candidate email? Can you flag to Maeve/Myles/James? @JR-G

**Review by tomas-stefano — APPROVED** [review]
Looks great! 👽 dance! (But don't merge until you confirm the candidate email :) )

File: app/services/decline_offer_by_default.rb (hunk 1 above)
Comment (avinhurry) [reply app/services/decline_offer_by_default.rb:30]: Yes I did @tomas-stefano, will edit my comment! 

@MaeveRoseveare @MylesJarvis @JR-G ☝🏼 is the plan to keep the DBD candidate email in?

Comment (MaeveRoseveare) [reply app/services/decline_offer_by_default.rb:30]: That is the plan for next cycle, as DBD will not exist any more. These should still remain functioning for this cycle though. We had a separate [ticket for the candidate emails](https://trello.com/c/mPBt2KOJ/483-ca-emails-remove-decline-by-default-dbd-emails) - will let Myles and James advise on what's best in terms of doing candidate + provider together!

We plan to replace these with new emails to nudge candidates to accept offers which I'm currently writing.

Comment (avinhurry) [reply app/services/decline_offer_by_default.rb:30]: Thanks @MaeveRoseveare! I'll merge this in tomorrow morning. 

Just found the card for removing the candidate DBD emails: https://trello.com/c/mPBt2KOJ/483-ca-emails-remove-decline-by-default-dbd-emails

Two for the price of one 🙌🏼

**Comment by avinhurry** [comment]
[CA emails: Remove Decline By Default (DBD) emails](https://trello.com/c/mPBt2KOJ/483-ca-emails-remove-decline-by-default-dbd-emails)

File: app/services/decline_offer_by_default.rb (hunk 1 above)
Comment (JR-G) [reply app/services/decline_offer_by_default.rb:30]: Yup – @MaeveRoseveare  has covered it & happy for both to go in together!

**Review by JR-G — APPROVED** [review]
🙇

**Pull request merged**